sdk/fd_create/prog.c

```diff
@@ -0,0 +1,80 @@
+// This test program is not ready to be released!
```

Comment:
Also: can you add our standard copyright/licensing header to the top of this file?

Reply:
Done.